src/content/guides/asset-management.md

```diff
@@ -59,16 +61,19 @@ __webpack.config.js__
 +       {
 +         test: /\.css$/,
 +         use: [
-+           'style-loader',
++           MiniCssExtractPlugin.loader,
```

Comment:
This won't work with hot reloading, at least not yet.

Reply:
Do you suggest we add a note that hot reloading wont work for now with MiniCssExtractPlugin or is there a known workaround? @Kovensky

Reply:
The known workaround is to use `style-loader` 😅

Reply:
Oh, so we probably can skip updating this guide for now (asset-management) and leave it with `style-loader`?

Reply:
a redirect from `deleted` plugin is added for that, also the text of the page says why it came to replace ETWP: https://github.com/webpack-contrib/mini-css-extract-plugin

Otherwise works fine locally.

@Kovensky am i right we dont need to update asset-management guide for now?